internal/api/api.go

```diff
@@ -512,8 +513,9 @@ type GetReleasePromotionRequest struct {
 }
 
 type GetReleasePromotionResponse struct {
-	ID     string `json:"id" db:"id"`
-	Status string `json:"status" db:"status"`
+	ID      string `json:"id" db:"id"`
```

Comment:
not sure why/when we had `db` tags here but we should get rid of them, don't need them